### PR TITLE
Remove redundant HttpClient DisconnectHandler

### PR DESCRIPTION
**Pull request by otoolec**
Also fixes the confusion caused by milli vs sec

### CODE REVIEW

File: src/main/java/com/xjeffrose/xio/client/HttpClientConnector.java [hunk 1]
```diff
@@ -101,15 +98,11 @@ protected void initChannel(SocketChannel channel) throws Exception {
...
-            20000,
-            NO_WRITER_IDLE_TIMEOUT,
-            NO_ALL_IDLE_TIMEOUT,
-            TimeUnit.MILLISECONDS));
-        cp.addLast("idleDisconnectHandler", new IdleDisconnectHandler(
-            2000,
-            NO_WRITER_IDLE_TIMEOUT,
-            NO_ALL_IDLE_TIMEOUT));
+        cp.addLast("idleDisconnectHandler", new XioIdleDisconnectHandler(
+              20000,
```
Comment (otoolec) [inline src/main/java/com/xjeffrose/xio/client/HttpClientConnector.java]: I'm actually not sure if this is the right timeout.  2,000 seconds seemed like a long time.

Comment (xjdr) [inline src/main/java/com/xjeffrose/xio/client/HttpClientConnector.java]: updated in master with the following 

```java
       if (def.getClientIdleTimeout() != null) {
          cp.addLast("idleDisconnectHandler", new XioIdleDisconnectHandler(
              (int) def.getClientIdleTimeout().toMillis(),
              NO_WRITER_IDLE_TIMEOUT,
              NO_ALL_IDLE_TIMEOUT,
              TimeUnit.MILLISECONDS));
        }
```

Comment (otoolec) [inline src/main/java/com/xjeffrose/xio/client/HttpClientConnector.java]: I only see that having been done for [XioServerTransport](https://github.com/xjdr/xio/blob/master/src/main/java/com/xjeffrose/xio/server/XioServerTransport.java#L91).  This PR is for doing a similar change in HttpClientConnector.

Should XioClientConfig be modified to add a client idle timeout?

**Comment by xjdr** [comment]
Sorry, misread the filename. Merged

File: src/main/java/com/xjeffrose/xio/client/HttpClientConnector.java (hunk 1 above)
Comment (xjdr) [inline src/main/java/com/xjeffrose/xio/client/HttpClientConnector.java]: Yes, it probably should be. I'm not a big fan of static timeout values.

**Pull request merged**